### ISSUE TITLE
Inquiry about finetuning and pretraining PP-HumanSegV2 model

### ISSUE THREAD

**Issue by Si-Ru** [PaddlePaddle/PaddleSeg#2994]
### 问题确认 Search before asking

- [X] 我已经搜索过问题，但是没有找到解答。I have searched the question and found no related answer.

### 请提出你的问题 Please ask your question

Thank you for developing this excellent model. It has been very helpful to me in my studies. 
Now, I try to do finetune the Portrait Segmentation Model with PP-HumanSeg V2 using my own dataset and portrait14K.
I do have some questions about the model and training process, so I have created an issue to discuss them.

Firstly, I would like to know if the data used to pretrain the portrait human segmentation model is the same as the data used for the general human segmentation model.

Secondly, I understand that the pretrained dataset cannot be shared, but could you provide some information about the quantity of data, number of people, background, target pose, and the configuration used for training?

Thirdly, I am curious to know whether the parameters of some layers are fixed during both pretraining and finetuning.

Finally, I would appreciate more information about the batch size of 8 - 4 gpu. When learning batch size 8, the training loss fluctuated too much.

Thank you very much for your time.
Best Regards.

**Comment by juncaipeng** [comment]
Thanks for your interest in our work.
1. Yes, these two models use the same dataset for pretraining.
2. The pretrained dataset information：train 7.6w, val 0.9w, test 0.9w; the background and human pose are various.
3. These layers are not fixed in pretraining and finetuning.

**Comment by Si-Ru** [comment]
Thank you so much for your prompt response. 

I have one more question about the pretrain model,
that may use mobilenet_v3_large_1.0_ssld model introduced in https://paddleclas.readthedocs.io/en/latest/models/Mobile_en.html
I'm curious whether this model has been trained for object detection or classification.

Thanks & Regards.

**Comment by juncaipeng** [comment]
The model is trained on Imagenet dataset for classification, so it can be used as pretrained weight for other tasks, such as object detection and segmentation.

**Comment by Si-Ru** [comment]
Good! Thank you for your kind response. It has been helpful.

Thanks & Regards.

**Comment by langxuelang** [comment]
@Si-Ru 

I also sent an email to the official team requesting access to the PP-HumanSeg14K dataset. Unfortunately, I haven't received any response after waiting for quite some time.

Would it be possible for you to share the dataset with me? I would greatly appreciate your help.

My email address is: [gaotengxuelang@gmail.com](mailto:gaotengxuelang@gmail.com)
